client/src/components/Model.js

```diff
@@ -120,6 +120,7 @@ export const NOTE_TYPE = {
 }
 
 export const DEFAULT_CUSTOM_FIELDS_PARENT = "formCustomFields"
+export const NOTES_FIELD = "notes"
```

Comment:
I'd move this line to below the next line (kepping the custom fields consts together).